dsc/src/main.rs

```diff
@@ -26,7 +26,7 @@ fn main() {
     check_debug();
 
     // create subscriber that writes all events to stderr
-    let subscriber = tracing_subscriber::fmt().pretty().with_writer(std::io::stderr).finish();
+    let subscriber = tracing_subscriber::fmt().pretty()/*.with_max_level(Level::DEBUG)*/.with_writer(std::io::stderr).finish();
```

Comment:
We should remove this and just have the separate PR to enable setting logging level to `dsc`

Reply:
ok. Removed.